src/data/coreteam.json

```diff
@@ -0,0 +1,117 @@
+{
+ "slug": "69cf48da-9394-4b5f-b484-460b8d08464b",
```

Comment:
is this required for global data on core team?

Reply:
nope

Reply:
When not needed, I think its better to remove it @HerrAlpha @fahlisaputra ?

Reply:
i couldn't merge it to main branch

Reply:
ok @vzrenggamani, im gonna remove it

Reply:
i just remove it

Reply:
Done. Slug removed.